pyarkconf/templates/symposion/schedule/_edit_grid.html

```diff
@@ -0,0 +1,39 @@
+<table class="calendar table table-bordered">
```

Comment:
Was this all freshly written code or did you have a reference implementation from somewhere?